Polyhedron/demo/Polyhedron/include/CGAL/IO/read_surf_trianglemesh.h

```diff
@@ -101,10 +100,12 @@ void read_surf(std::istream& input, std::vector<Mesh>& output, std::vector<Mater
     }
   std::cout<<nb_patches<<" patch(es)"<<std::endl;
   metadata.resize(nb_patches);
-  for(int i=0; i < nb_patches; ++i)
+  output.resize(nb_patches);
+  const vertex_descriptor null_vertex = boost::graph_traits<Mesh>::null_vertex();
+  std::vector<vertex_descriptor> vertices(nb_vertices, null_vertex);
+  for(int i=0; i < nb_patches; /* i is incremented in the body */)
   {
-    std::map<std::size_t, vertex_descriptor> vertices;
```

Comment:
The next speedup is the removal of the `std::map` `vertices. Instead, I allocated once a big vector, and fill it with null vertex descriptors...